config/sampler_config.go

```diff
@@ -30,6 +30,13 @@ const (
 	NotIn          = "not-in"
 )
 
+const (
+	ComputedFieldPrefix               = "?."
+	NUM_DESCENDANTS     ComputedField = ComputedFieldPrefix + "NUM_DESCENDANTS"
+)
+
+type ComputedField string
```

Comment:
Really minor nit, but I'd prefer to put the type before the const that uses it.

Reply:
Yes, I agree that's better for readability. I will make that change